app.js

```diff
@@ -11,3 +11,171 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    Experimental content creation feature that allows users to add to an
+    existing story over the course of a day without spamming their
+    friends.
+    `,
+    featuredImage: '/images/Snapshoot-Portfolio1.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card2',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+
+  {
+    name: 'Facebook 360',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    Exploring the future of media in Facebook's first Virtual Reality
+    app; a place to discover and enjoy 360 photos and videos on Gear VR. `,
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card3',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+  {
+    name: 'Uber Navigation',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    A smart assistant to make driving more safe, efficient, and fun by
+    unlocking your most expensive computer: your car. `,
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card4',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+];
+
+function newCard(card) {
+  return `
+    <div class="card">
+        <img src="${card.featuredImage}" alt="" />
+        <div class="card-content">
+          <h2>${card.name}</h2>
+          <div class="list-1">
+            <p>${card.company}</p>
+            <ul class="field">
+              <li class="backend">${card.stack}</li>
+              <li class="year">${card.year}</li>
+            </ul>
+          </div>
+
+          <p>
+          ${card.description}
+          </p>
+          <ul class="languages">
+            <li>${card.tech1}</li>
+            <li>${card.tech2}</li>
+            <li>${card.tech3}</li>
+          </ul>
+          <div id="btn-curser">
+          <button onclick="openModal('${card.modalId}')" class="btn modal-btn" id="${card.modalId}">See Project</button>
+            
+          </div>
+        </div>
+      </div>
+  `;
+}
+function openModal(modalId) {
+  const btnOpen = document.querySelector(`#${modalId}`);
+  const modalOverlay = document.querySelector('.modal-overlay');
+  btnOpen.addEventListener('click', () => {
+    modalOverlay.classList.add('show-container');
+  });
+  cards.forEach((card) => {
+    if (card.modalId === modalId) {
+      const x = document.querySelector('.modal-container');
+      x.innerHTML = `
+  <div class="card">
```

Comment:
- [ ] Your pop-up is not scrollable. I can't see all the information on the pop-up. Please, add a scrollbar to your pop-up or fix the height of the pop-up so your pop-up fits all mobile and desktop screens. 
- Your design:
<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/157998777-55516f88-6123-4001-bc1b-0fd5331285ca.png">